symplyphysics/laws/electricity/energy_two_charges_from_distance_and_relative_permittivity.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, pi)
+from sympy.physics.units import electric_constant
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    dimensionless,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Potential energy is a function of the state of the system. The zero value is taken when the charges are infinitely
+## far from each other. Note also that this is the energy of the entire system, the energy of interaction, so it makes
+## no sense to say that some part of this energy belongs to one of the charges.
+
+## Law is: W = q1 * q2 / (4 * pi  * e0 * e * r), where
+## W -energy of interaction of charges,
+## q1 - first charge,
+## q2 - second charge,
+## e0 - electric constant,
+## r - distance between charges.
+
+energy = Symbol("energy", units.energy)
+
+relative_permittivity = Symbol("relative_permittivity", dimensionless)
+distance = Symbol("distance", units.length)
+charge_1 = Symbol("charge_electron", units.charge)
```

Comment:
This law is not limited to electrons.
Also give different names to different symbols.